src/data_aggregator.ts

```diff
@@ -1,42 +1,45 @@
-import { strict as assert } from 'assert'
-import BigNumber from 'bignumber.js'
-import Logger from 'bunyan'
 import * as aggregators from './aggregator_functions'
-import { OracleApplicationConfig } from './app'
+
+import {
+  AggregationMethod,
+  CurrencyPairBaseQuote,
+  Exchange,
+  OracleCurrencyPair,
+  PromiseStatus,
+  SettledPromise,
+  allSettled,
+} from './utils'
 import { ExchangeAdapter, ExchangeAdapterConfig } from './exchange_adapters/base'
+import {
+  ExchangePriceSource,
+  ExchangePriceSourceConfig,
+  OrientedAdapter,
+  OrientedExchangePair,
+} from './exchange_price_source'
+import { PriceSource, WeightedPrice } from './price_source'
+
+import { AlphavantageAdapter } from './exchange_adapters/alphavantage'
+import BigNumber from 'bignumber.js'
 import { BinanceAdapter } from './exchange_adapters/binance'
 import { BinanceUSAdapter } from './exchange_adapters/binance_us'
-import { OKXAdapter } from './exchange_adapters/okx'
+import { BitMartAdapter } from './exchange_adapters/bitmart'
+import { BitgetAdapter } from './exchange_adapters/bitget'
 import { BitsoAdapter } from './exchange_adapters/bitso'
+import { BitstampAdapter } from './exchange_adapters/bitstamp'
 import { BittrexAdapter } from './exchange_adapters/bittrex'
 import { CoinbaseAdapter } from './exchange_adapters/coinbase'
-import { OKCoinAdapter } from './exchange_adapters/okcoin'
-import { NovaDaxAdapter } from './exchange_adapters/novadax'
+import { GeminiAdapter } from './exchange_adapters/gemini'
 import { KrakenAdapter } from './exchange_adapters/kraken'
 import { KuCoinAdapter } from './exchange_adapters/kucoin'
-import { BitstampAdapter } from './exchange_adapters/bitstamp'
+import Logger from 'bunyan'
```

Comment:
nit
maybe we can move this import to the top and not in between the aggregators

Reply:
This was not me but the linter, so I will just end up moving it back